.github/CONTRIBUTING.md

```diff
@@ -73,22 +73,27 @@ resource.TestStep{
 },
 ```
 
-## Running a local copy of the provider
-If you are building Terraform from source with a Google provider built from source, Terraform will automatically use the
+## Instructing terraform to use a local copy of the provider
+There are three different consumption strategies you can utilize
```

Comment:
nit: should we add some spaces between headers and text here (and below)?

Reply:
Changed header to bullet list. looks slightly better but not sure what I like more. I tried adding the spacing below/above this text. Looked kind of funky

Reply:
unless you mean adding spacing purely for the reader of the raw readme files

Reply:
resolving for now. feel free to reopen if you have strong opinion.

Reply:
Oh, I just meant the raw markdown!